airbyte-integrations/connectors/source-okta/source_okta/schemas/logs.json

```diff
@@ -32,7 +32,8 @@
             "LDAP",
             "FEDERATION",
             "SOCIAL",
-            "FACTOR_PROVIDER"
+            "FACTOR_PROVIDER",
+            null
```

Comment:
I want to highlight here for the connectors platform team. I am new to python, I learned that `null` is `None` in python. In fact, `null` is not recommended in the enum array. If I don't add this line, the acceptance test will throw an error because the value is null in one of the logs. 

Though the value type is `["string", "null"]`, it doesn't accept `null` or `None` 🤷 . Specifically, the method from `python-jsonschema` throws an error:

https://github.com/python-jsonschema/jsonschema/blob/main/jsonschema/_validators.py#L279-L285


```
ERROR    root:test_core.py:299 
The logs stream has the following schema errors:
None is not one of ['OKTA_AUTHENTICATION_PROVIDER', 'ACTIVE_DIRECTORY', 'LDAP', 'FEDERATION', 'SOCIAL', 'FACTOR_PROVIDER']

Failed validating 'enum' in schema['properties']['authenticationContext']['properties']['authenticationProvider']:
    {'enum': ['OKTA_AUTHENTICATION_PROVIDER',
              'ACTIVE_DIRECTORY',
              'LDAP',
              'FEDERATION',
              'SOCIAL',
              'FACTOR_PROVIDER'],
     'type': ['string', 'null']}
```

I am pretty sure the acceptance test is not 100% at the current `master` branch. I want to achieve it, so I apply this duck tape.